datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -105,9 +116,12 @@ def get_available_tox_envs(check, sort=False, e2e_only=False, e2e_tests_only=Fal
         tox_command = 'tox --listenvs'
 
     with chdir(path_join(get_root(), check)):
-        output = run_command(tox_command, capture='out').stdout
+        output = run_command(tox_command, capture='out')
+
+    if output.code != 0:
+        abort(output.stderr + '\n' + output.stdout)
```

Comment:
To more easily see errors

```suggestion
        abort(output.stdout + '\n' + output.stderr)
```